utils/snapshotHub.ts

```diff
@@ -0,0 +1,319 @@
+import axios from "axios";
+
+export enum VoteChoices {
+  No = "No",
+  Yes = "Yes",
+}
+
+export enum SnapshotType {
+  draft = "draft",
+  proposal = "proposal",
+  vote = "vote",
+}
+
+// Ordered vote choices. Do not change the indexes!
+export type CoreProposalVoteChoices = [VoteChoices.Yes, VoteChoices.No];
+
+export type SnapshotCoreProposalPayloadData = {
+  /**
+   * Readable name of the proposal
+   */
+  name: string;
+  /**
+   * Readable description of the proposal
+   */
+  body: string;
+  /**
+   * Choices for Snapshot vote
+   */
+  choices: CoreProposalVoteChoices;
+  /**
+   * Stringifiable metadata for the proposal, e.g. `private: 1`
+   */
+  metadata: Record<string, any>;
+};
+
+export type SnapshotProposalPayloadData = {
+  /**
+   * Vote start timestamp string (in seconds)
+   */
+  start: string;
```

Comment:
in milliseconds (unix)

Reply:
I think I have it wrong for type, it should be `number`. If you take a look at the API response, it's using unix seconds `number`.

Reply:
https://snapshot-hub.dev.thelao.io/api/thelao/proposals